src/CookieCollection.php

```diff
@@ -0,0 +1,298 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Web;
+
+use ArrayAccess;
+use ArrayIterator;
+use Closure;
+use Countable;
+use DateTimeImmutable;
```

Comment:
Not used.

Reply:
What do you mean by not used? `CookieCoollection` implements `Countable`, it uses `ArrayIterator` in `getIterator` metho and also uses `Closure` in `exists` method.

`DateTimeImmutable` and `Psr\Http\Message\ServerRequestInterface` are not used, I just forgot to remove `use` statements. I still have some pending changes, and will remove them either.

Reply:
Yes, I've meant only the lines I've commented on.